package_arcayi_ArduinoCore-GD32_index.json

```diff
@@ -1,15 +1,23 @@
 {
   "packages": [
     {
+      "name": "GD32Community-fork",
+      "maintainer": "arcayi",
+      "websiteURL": "https://github.com/arcayi/ArduinoCore-GD32/",
```

Comment:
For the merge, this should be reverted to the original one.

Reply:
fixed the commit files